src/main.js

```diff
@@ -1 +1,17 @@
-/* Initial script */
\ No newline at end of file
+/* Initial script */
+
+console.log(Vector.Create("(2p, 1{3})j; (1, 0)d;"));
+console.log(Vector.Create("(1, 1{3}+)"));
+
+var queen = new Piece();
+queen.setMoveVectors(Vector.Create("(1, 0)+;(0, 1)+;(1, 1)+"))
```

Comment:
If we're going with `Piece.Create` this would be a great place to use it.

Reply:
Nope. This is testing to validate that the other function in Piece assign the values correctly. Main.js is a test file right now. Hence Piece.Create() is used and tested later in the file.

Reply:
Ah yes I see that now.